.github/workflows/eslint.yml

```diff
@@ -13,5 +13,5 @@ jobs:
         with:
           eslintrc: false # Set this to true if you want to use your own .eslintrc rules
           extensions: 'js'
-          auto-fix-before-test: false # Set this to true if want to format and fix all the lint issue before testing
+          auto-fix-before-test: true # Set this to true if want to format and fix all the lint issue before testing
```

Comment:
So the point of this is to point out potential quality issues with your code, for example that you're missing semicolons at the end of some lines. Please don't just set it to automatically "fix" things.